Public/Src/Tools/BuildXL.Tools.CredentialProvider/BuildXL.Tools.CredentialProvider.csproj

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Can we build this tool with BuildXL instead and then delete all these MSBuild files?

Reply:
Also, do we have to check in the `BuildXL.Tools.CredentialProvider.1.0.0.nupkg` binary?